src/main/resources/locale_es_ES.properties

```
@@ -0,0 +1,196 @@
+command.thankwords.sub.check.reference=Mensaje de referencia: [Jump](%URL%)
+command.thankwords.sub.check.result=Donador: %DONATOR%\nReceptor: %RECEIVER%
+command.thankwords.sub.list=Lista de todos los patrones
+command.thankwords.sub.list.list=Los siguientes patrones están activos:
+command.thankwords.sub.loadDefault=Cargar palabras de agradecimiento por defecto para un idioma.
+command.thankwords.sub.loadDefault.added=Añadir las siguientes palabras de agradecimiento:
+command.thankwords.sub.loadDefault.available=Idiomas disponibles:
+command.thankwords.sub.remove=Eliminar un patrón
+command.thankwords.sub.remove.removed=Eliminar patrón %PATTERN%.
+conversation.cancel=Cancelar
+conversation.canceled=Conversación cancelada.
+conversation.exit=salir
+conversation.finished=Conversación terminada.
+conversation.inProgress=Ya hay una conversación en curso en este canal. Puedes salir de esta conversación escribiendo $conversation.exit$ o $conversation.cancel$.
+conversation.start=Inicio de una nueva conversación.\NPuede abandonar la conversación en cualquier momento escribiendo $conversation.exit$ o $conversation.cancel$.
+error.commandNotFound=Comando no encontrado.
+error.invalidArguments=Argumentos no válidos
+error.invalidChannel=Canal no válido
+error.invalidMessage=Mensaje no válido
+error.invalidNumber=Número no válido
+error.invalidRegex=Patrón regex inválido
+error.invalidRole=Rol inválido
+error.missingPermission=Me falta el permiso %PERM% para que funcione correctamente.
+error.missingPermissionChannel=Esto se debe a la configuración de los permisos en el canal %CHANNEL%.
+error.missingPermissionGuild=Esto se debe a la configuración de permisos globales.
+error.noValidChannel=No se han proporcionado canales válidos.
+error.notABoolean=%INPUT% no es un booleano (true|false).
+error.notANumber=%INPUT% no es un número.
+error.notYourEmbed=¡Este no es su embed!
+error.prefixTooLong=El prefijo sólo puede tener 3 caracteres.
+error.roleAccess=No se puede interactuar con el rol %ROLE%. Por favor, ajuste la jerarquía.
+error.userNotFound=No se ha podido encontrar este usuario.
+listener.messages.request.descr=Hey, acabas de agradecer a alguien, ¿quieres agradecer a alguien en particular? Si es así, haz clic en el botón de abajo.
+listener.messages.request.descrThank=Gracias por dar reputación a alguien. Puedes dar %MORE% más reputación.
+listener.messages.request.title=Gracias confirmación
+listener.reaction.confirmation=%DONOR% ha dado reputación a %RECEIVER%.
+listener.reaction.removal=La reputación dada por %DONOR% fue eliminada.
+message.welcome=Gracias por invitarme.
+messages.destruction=Este mensaje se autodestruirá en %MIN% minuto.
+thankType.answer=Respuesta
+thankType.fuzzy=Fuzzy
+thankType.mention=Mención
+thankType.reaction=Reacción
+word.continue=Continuar
+word.done=Hecho
+word.skip=Saltar
+words.code=Código
+words.jumpMarker=[%TARGET%](%URL%)
+words.language=Langauge
```

Looks untranslated

DeepL suggestion:
```suggestion
words.language=Lengua
```